package.json

```diff
@@ -12,93 +12,44 @@
     "build": "turbo run build tsc",
     "build-storybook": "storybook build",
     "build:docs": "npx ts-node ./scripts/parse-tsdocs.ts",
-    "lint": "npm-run-all --parallel lint:*",
-    "lint:prettier": "prettier --check \"**/*.{js,ts,tsx,json,md,yml}\"",
-    "lint:eslint": "eslint \"**/*.{js,ts,tsx}\"",
+    "lint": "lg-lint",
     "chromatic": "npx chromatic",
     "clean": "yarn clean:builds && yarn clean:modules",
     "clean:builds": "rm -rf packages/*/dist packages/*/tsconfig.tsbuildinfo ./tsconfig.tsbuildinfo packages/**/stories.js",
     "clean:modules": "rm -rf node_modules packages/*/node_modules ",
-    "fix": "npm-run-all --parallel fix:*",
-    "fix:prettier": "prettier --write \"**/*.{js,ts,tsx,json,md,yml}\"",
-    "fix:eslint": "eslint \"**/*.{js,ts,tsx}\" --fix",
+    "fix": "lg-lint --fix",
     "link": "./scripts/link.sh",
     "link:all": "lerna exec -- yarn link",
     "prepublishOnly": "yarn fix && yarn test && yarn build && yarn prepublish:ts-downlevel",
     "prepublish:ts-downlevel": "rm -rf packages/*/dist/ts3.4 && echo packages/*/dist | NODE_NO_WARNINGS=1 xargs -I{} -n1 -P8 downlevel-dts {} {}/ts3.4",
     "publish": "npm-run-all --serial publish:*",
     "publish:packages": "yarn changeset publish --public",
-    "slackbot": "npx ts-node ./scripts/slackbot",
+    "slackbot": "slackbot",
     "start": "storybook dev -p 9001 --no-version-updates",
-    "test": "npx ts-node ./scripts/test.ts",
-    "test:local": "yarn test",
-    "test:ssr": "yarn test --ssr",
+    "test": "lg-test",
+    "test:ssr": "lg-test --ssr",
```

Comment:
this script not working either, but I have a feeling we'll be able to remove. I'm not entirely convinced this is providing enough value for the maintenance burden introduced.

```
☁  leafygreen-ui [integration-shared-config] yarn test:ssr
yarn run v1.22.19
$ lg-test --ssr
(node:79659) Warning: Accessing non-existent property 'setupFiles' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
TypeError: defaultConfig.setupFiles is not iterable
    at Object.<anonymous> (/Users/brooke.yalof/mdb/leafygreen-ui/tools/test/config/ssr/jest.config.js:6:22)
    at Module._compile (node:internal/modules/cjs/loader:1196:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1250:10)
    at Module.load (node:internal/modules/cjs/loader:1074:32)
    at Function.Module._load (node:internal/modules/cjs/loader:909:12)
    at Module.require (node:internal/modules/cjs/loader:1098:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at readConfigFileAndSetRootDir (/Users/brooke.yalof/mdb/leafygreen-ui/node_modules/jest-config/build/readConfigFileAndSetRootDir.js:119:22)
    at readConfig (/Users/brooke.yalof/mdb/leafygreen-ui/node_modules/jest-config/build/index.js:210:65)
    at readConfigs (/Users/brooke.yalof/mdb/leafygreen-ui/node_modules/jest-config/build/index.js:406:32)
✨  Done in 0.28s.
```